benchmarks/test_ia_profile.py

```diff
@@ -0,0 +1,94 @@
+    k_arr = np.geomspace(1E-3, 1e2, 128)  # For evaluating
+    a_arr = np.linspace(0.1, 1, 16)
+
+    cM = ccl.halos.ConcentrationDuffy08(mass_def="200m")
+    nM = ccl.halos.MassFuncTinker08(mass_def="200m")
+    bM = ccl.halos.HaloBiasTinker10(mass_def="200m")
+    hmc = ccl.halos.HMCalculator(mass_function=nM,
+                                 halo_bias=bM,
+                                 mass_def="200m", nM=64)
+
+    sat_gamma_HOD = ccl.halos.SatelliteShearHOD(concentration=cM,
+                                                mass_def='200m')
+    NFW = ccl.halos.HaloProfileNFW(concentration=cM, truncated=True,
+                                   fourier_analytic=True, mass_def='200m')
+
+    with pytest.warns(ccl.CCLDeprecationWarning):
+        pk_GI_1h = ccl.halos.halomod_Pk2D(cosmo, hmc, NFW,
+                                          normprof1=True,
+                                          normprof2=True,
+                                          prof2=sat_gamma_HOD,
+                                          get_2h=False,
+                                          lk_arr=np.log(k_arr),
+                                          a_arr=a_arr)
+    with pytest.warns(ccl.CCLDeprecationWarning):
+        pk_II_1h = ccl.halos.halomod_Pk2D(cosmo, hmc, sat_gamma_HOD,
+                                          normprof1=True,
+                                          normprof2=True,
+                                          get_2h=False,
+                                          lk_arr=np.log(k_arr),
+                                          a_arr=a_arr)
+
+    z_arr = np.linspace(0., 3., 256)
+    z0 = 0.1
+    pz = 1. / (2. * z0) * (z_arr / z0) ** 2. * np.exp(-z_arr / z0)
+
+    b_IA = np.ones(len(z_arr))  # A_IA = 1 in the NLA model
+    ia_tracer = ccl.WeakLensingTracer(cosmo, dndz=(z_arr, pz), has_shear=False,
+                                      ia_bias=(z_arr, b_IA), use_A_ia=False)
+    wl_tracer = ccl.WeakLensingTracer(cosmo, dndz=(z_arr, pz))
+
+    cl_GI_1h = ccl.angular_cl(cosmo, wl_tracer, ia_tracer, l_arr,
+                              p_of_k_a=pk_GI_1h)
+    cl_II_1h = ccl.angular_cl(cosmo, ia_tracer, ia_tracer, l_arr,
+                              p_of_k_a=pk_II_1h)
+
+    assert np.all(np.fabs(cl_GI_1h / cl_GI_benchmark - 1) < 0.01)
+    assert np.all(np.fabs(cl_II_1h / cl_II_benchmark - 1) < 0.02)
+
+    # Below we are benchmarking the normalization factor of the
+    # delta-E power spectrum Eq. (17) in Fortuna et al. 2021.
+    # Specifically, the term f_s(z)*<N_s|M>/n_s(z).
+    mass_benchmark, norm_benchmark = np.loadtxt(
+        './benchmarks/data/IA_halomodel_norm_term.dat', unpack=True)
+    cosmo = ccl.Cosmology(Omega_c=0.25, Omega_b=0.05, Omega_k=0,
+                          sigma8=0.81, n_s=0.96, h=1.)
+    cosmo.compute_sigma()
+    Ns_M_mean = sat_gamma_HOD._Ns(mass_benchmark, 1.)
+    norm = Ns_M_mean / sat_gamma_HOD.get_normalization(cosmo, 1, hmc)
```

Comment:
The normalisation here divides by the `get_normalization` while in [my PR](https://github.com/LSSTDESC/CCL/blob/3ed354cbfcd2cfbb51f8ce6ccf4f4bd1d480d998/benchmarks/test_haloprofile.py#L264) I multiply. Has the normalization changed in v3 for profiles?

Reply:
yes, the normalization is now the thing you divide by